build_tools/docker/mmperf/Dockerfile

```diff
@@ -0,0 +1,110 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# An image for running mmperf: https://github.com/mmperf/mmperf.
+#
+# mmperf benchmarks matrix-multiplication workloads on IREE and various backends
+# such as OpenBLAS, MKL, TVM, Halide, CuBLAS, etc.
+#
+# These backends are included either in this image or as a submodule in the
+# mmperf repo. Later versions of Clang, LLVM, Python and Ubuntu are needed
+# to satisfy the dependency requirements of the backends.
+
+FROM ubuntu:22.04
```

Comment:
We use pinned hashes for everything. Otherwise, the build result heavily depends on what the builder happens to have on their local machine

Reply:
Pinned to a hash now